vllm/config/model.py

```diff
@@ -1197,10 +1169,10 @@ def get_sliding_window(self) -> int | None:
         return getattr(self.hf_text_config, "sliding_window", None)
 
     def get_vocab_size(self) -> int:
```

Comment:
With this PR, can we get rid of functions like `get_vocab_size` and directly use `model.model_arch_config.vocab_size`? It would be great if we can do it in a future PR to reduce the number of hops to get these config values.